file.c

```diff
@@ -8236,7 +8249,8 @@ save2tmp(URLFile uf, char *tmpf)
 
 	buf = NewWithoutGC_N(char, SAVE_BUF_SIZE);
 	while ((count = ISread_n(uf.stream, buf, SAVE_BUF_SIZE)) > 0) {
-	    if (fwrite(buf, 1, count, ff) != count) {
+	    int c = fwrite(buf, 1, count, ff);
+	    if (c != count) {
```

Comment:
Why this change?

Reply:
My guess is that I was going to use c again, but then I didn't and forgot about the line :/
I will change it back; thank you for your input.